theme/scripts/productDescription.ts

```diff
@@ -0,0 +1,83 @@
+export default(function() {
+	const ProductDescription = {
+		SELECTORS: {
+			tab: "[data-product-description-tab]",
+      content: "[data-product-description-content]",
+      tabsWrapper: ".ProductDescription__tabs",
+      nav: ".Nav"
+		},
+    CLASSES: {
+      activeTab: "ProductDescription__button--active"
+    },
+
+		init() {
+			const tabs = [].slice.call(
+        document.querySelectorAll(ProductDescription.SELECTORS.tab)
+      );
+
+      const tabbedContent = [].slice.call(
+        document.querySelectorAll(ProductDescription.SELECTORS.content)
+      );
+
+      const tabsWrapper: HTMLElement = document.querySelector(ProductDescription.SELECTORS.tabsWrapper);
+
+      const navs = [].slice.call(
+        document.querySelectorAll(ProductDescription.SELECTORS.nav)
+      );
+
+			if (!tabs || !tabbedContent || !tabsWrapper || !navs) return;
+
+      let height = ProductDescription.getStickyHeight(tabsWrapper, navs);
+      window.addEventListener('resize', () => {
+        height = ProductDescription.getStickyHeight(tabsWrapper, navs);
+      })
+      
+      window.addEventListener('scroll', () => {
+        window.requestAnimationFrame(() => {
+          ProductDescription.scrollHandler(tabs, tabbedContent, height);
+        });
+      })
+		},
+
+    getStickyHeight(tabsWrapper: HTMLElement, navs: HTMLElement[]): number {
+      const tabsWrapperHeight = tabsWrapper.offsetHeight;
+      const activeNav = navs.find((elem) => {
+        if (elem.offsetHeight !== 0) return elem;
+      })
+      return tabsWrapperHeight + activeNav.offsetHeight + 16;
+    },
+
+    toggleActiveStyling(isActive: boolean, tab: HTMLElement) {
+      if (isActive) {
+        tab.classList.add(
+          ProductDescription.CLASSES.activeTab
+         );
+      } else {
+        tab.classList.remove(
+          ProductDescription.CLASSES.activeTab
+        );
+      }
+    },
+
+		scrollHandler(tabs: HTMLElement[], tabbedContent: HTMLElement[], height: number) {
+      tabbedContent.forEach((elem, index) => {
+        const offsetTop = elem.offsetTop;
```

Comment:
Are we restricting tabbedContent to only 3? If so, we should probably ensure some type of validation at the accentuate level.

Reply:
Is there something in the code that makes it seem like only three? I kept it flexible so it could be as many tabs as Elie wants.

**Short answer: Can be more than 3**, but I never thought about the case where there is only one and would be no need for a sticky header? I'll ask Jake.

Reply:
@sepowitz when I talked to Jake, he mentioned the number of tabs should be flexible in case they want more than 3. BUT it doesn't look great when the nav content gets really long and needs to side scroll. 🤷🏻‍♀️ What do you think @uditdesai?

Reply:
Agreed that the default chrome scrollbar horizontally doesn't look great. But, for smaller screen sizes, it's going to have to happen. Maybe keep the font size smaller and set a max to avoid the scroll for most screen sizes?

Reply:
Apologies! I misread the `scrollHandler` method. Let's hide the scroll bar with css, and ask Jake for notes in QA.